orchagent/switch/swschema.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
```

Comment:
@prsunny switch schema is currently used in two places: helper and capabilities sections. It is a convenient way to manage things like that. It has nothing to do with capabilities section only. It can't be done in the way suggested.

Reply:
ok